src/xclim/data/fr.json

```
@@ -1475,5 +1475,11 @@
     "long_name": "Intensité du cycle de l'eau",
     "description": "Indice {freq} de l'intensité du cycle de l'eau, défini comme la somme de la précipitation et de l'évapotranspiration.",
     "abstract": "la somme de la précipitation et de l'évapotranspiration."
+  },
+  "ANTECEDENT_PRECIPITATION_INDEX": {
+    "title": "Indice de précipitations antécédentes",
+    "long_name": "Indice de précipitations antécédentes",
+    "description": "Calculer la somme pondérée des valeurs de précipitations quotidiennes pour une fenêtre ({window}) et un exposant de pondération ({p_exp}).",
+    "abstract": "Calcul de la somme pondérée des précipitations quotiediennes sur la base d'une fenêtre et d'un exposant de pondération."
```

```suggestion
    "title": "Indice de précipitation antérieure",
    "long_name": "Indice de précipitation antérieure",
    "description": "Somme pondérée des précipitations quotidiennes, sur une fenêtre de {window} jours et le base exponentielle {p_exp}.",
    "abstract": "Calcul de la somme pondérée des précipitations quotidiennes sur la base d'une fenêtre et d'une base exponentielle."
```
I found an official traduction here : https://www.btb.termiumplus.gc.ca/tpv2alpha/alpha-eng.html?lang=eng&i=1&srchtxt=ANTECEDENT+PRECIPITATION+INDEX&index=ent&codom2nd_wet=1#resultrecs
Also in WMO's glossary.